src/basicfuns.jl

```diff
@@ -441,3 +441,69 @@ $(SIGNATURES)
 Compute the complementary double exponential, `1 - exp(-exp(x))`.
 """
 cexpexp(x) = -_expm1(-exp(x))
+
+#=
+this uses the identity:
+
+log(logistic(x)) = -log(1 + exp(-x))
+=#
+"""
+$(SIGNATURES)
+
+Return `log(logistic(x))`, computed more carefully and with fewer calls
+than the naive composition of functions.
+
+Its inverse is the [`logitexp`](@ref) function.
+"""
+loglogistic(x::AbstractFloat) = -log1pexp(-x)
```

Comment:
```suggestion
loglogistic(x::Real) = -log1pexp(-x)
```
should be sufficient here (the line below should be removed, see https://github.com/JuliaStats/LogExpFunctions.jl/pull/82/files#r1589704384).